### PR TITLE
JBPM-5467: Stunner - UXD palette integration

### PR DESCRIPTION
**Pull request by pefernan**
Added new Palette with the UXD designs

### CODE REVIEW

**Comment by pefernan** [comment]
@romartin can you review it?

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/palette/CaseManagementBS3PaletteViewFactory.java [hunk 1]
```diff
@@ -21,22 +21,23 @@
...
+import org.kie.workbench.common.stunner.client.widgets.palette.factory.icons.bs3.BS3IconRenderer;
 import org.kie.workbench.common.stunner.cm.CaseManagementDefinitionSet;
 import org.kie.workbench.common.stunner.core.client.api.ShapeManager;
 
 @ApplicationScoped
-public class CaseManagementBS3PaletteViewFactory extends BindableBS3PaletteGlyphViewFactory<Icon> {
+public class CaseManagementBS3PaletteViewFactory extends BindableBS3PaletteGlyphViewFactory {
 
-    private final static Map<String, Icon> CATEGORY_VIEWS = new HashMap<String, Icon>() {{
+    private final static Map<String, IconResource> CATEGORY_RERNDERERS_SETTINGS = new HashMap<String, IconResource>() {{
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/palette/CaseManagementBS3PaletteViewFactory.java]: Typo (don't consider this an in depth review at this stage - I was just being nosy re: Case Management stuff)

Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/palette/CaseManagementBS3PaletteViewFactory.java]: I'll review tomorrow.

Comment (pefernan) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/palette/CaseManagementBS3PaletteViewFactory.java]: @manstis I'll wait your full review tomorrow ;)

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
@romartin yes it was related with the changes on the task-user.svg. Just modified it again and now works fine.

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
jenkins retest this

**Comment by romartin** [comment]
hey @manstis , I have tested the palette with a CM diagram and it seems that it works fine too.

**Pull request merged**